distros/tutorials/gradle.properties

```diff
@@ -0,0 +1,3 @@
+# The tutorials distro is full of our various developer tutorials
+extraModules=CoreSampleGameplay,Sample,Tutorial*
+zipFileName=TerasologySample.zip
```

Comment:
Wrong filename?
```suggestion
zipFileName=TerasologyTutorials.zip
```

Reply:
Fair, although I'm not sure anybody would ever really want to download this zip, if we ever even built such a thing :-)

Mostly made the distro for testing with `gw module init tutorials` - wouldn't hurt to be more accurate tho 👍